tmt/steps/execute/upgrade.py

```diff
@@ -10,50 +10,53 @@
 from tmt.steps.execute.internal import ExecuteInternal
 
 STATUS_VARIABLE = 'IN_PLACE_UPGRADE'
-BEFORE_UPGRADE_SUFFIX = 'old'
-DURING_UPGRADE_SUFFIX = 'upgrade'
-AFTER_UPGRADE_SUFFIX = 'new'
-UPGRADES_DIRECTORY = 'upgrades'
+BEFORE_UPGRADE_PREFIX = 'old'
```

Comment:
Oh good catch with the names, while going back and forth between prefixing and suffixing, I forgot to update it in here